docs/EN/Resources/clinician-guide-to-AndroidAPS.md

```diff
@@ -0,0 +1,101 @@
+
+### How A DIY Closed Loop Works
+
+Without a closed loop system, a person with diabetes gathers data from their pump and CGM, decides what to do, and takes action.
+
+With automated insulin delivery, the system does the same thing: it gathers data from the pump, CGM, and anywhere else information is logged (such as Nightscout), uses this information to do the math and decide how much more or less insulin is needed (above or below the underlying basal rate), and uses temporary basal rates to make the necessary adjustments to keep or eventually bring BGs into target range.
+
+If the device running AndroidAPS breaks or goes out of range of the pump, once the latest temporary basal rate ends, the pump falls back to being a standard pump with the usual basals runnning. 
+
+### How data is gathered:
+
+With AndroidAPS, an Android device runs a special app to do the math, the device communicates using Bluetooth with a supported pump. AndroidAPS can communicate with other devices and to the cloud via wifi to gather additional information, and to report to the patient, caregivers, and loved ones about what it’s doing and why.
+
+The Android device needs to:
+* communicate with the pump and read history - what insulin has been delivered
+* communicate with the CGM (either directly, or via the cloud) - to see what BGs are/have been doing
+
+When the device has collected this data, the algorithm runs and does the decision-making math based on the settings (ISF, carb ratio, DIA, target, etc.). If required, it then issues commands to the pump to modify insulin distribution.
+
+It will also gather any information about boluses, carbohydrate consumption, and temporary target adjustments from the pump or from Nightscout.
+
+### How does it know what to do? 
+
+The open source software is designed to make it easy for the device to do the work people used to do (in manual mode) to calculate how insulin delivery should be adjusted. It first collects data from all the supporting devices and from the cloud, prepares the data and runs the calculations, makes predictions of what BGs will be expected to do in different scenarios, and calculates the needed adjustments to keep or bring BG back into target range. Next it attempts to communicate and send any necessary adjustments to the pump. Then it reads the data back, and does it over and over again. 
+
+AndroidAPS is designed to transparently track all input data it gathers, the resulting recommendation, and any action taken. It is therefore easy to answer the question at any time, “why is it doing X?” by reviewing the logs.
+
+### Examples of AndroidAPS algorithm decision making:
+
+AndroidAPS makes multiple predictions (based on settings, and the situation) representing different scenarios of what might happen in the future. In Nightscout, these are displayed as “purple lines”. In the logs, it will describe which of these predictions and which time frame is driving the necessary actions.
```

Comment:
```suggestion
AndroidAPS uses the same core algorithm and feature set as OpenAPS. The algorithm makes multiple predictions (based on settings, and the situation) representing different scenarios of what might happen in the future. In Nightscout, these are displayed as “purple lines”. In the logs, it will describe which of these predictions and which time frame is driving the necessary actions.
```